src/main/java/org/opensearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -910,21 +910,21 @@ public List<Setting<?>> getSettings() {
                 AnomalyDetectorSettings.MAX_RUNNING_ENTITIES_PER_DETECTOR_FOR_HISTORICAL_ANALYSIS,
                 AnomalyDetectorSettings.MAX_CACHED_DELETED_TASKS,
                 // rate limiting
-                AnomalyDetectorSettings.CHECKPOINT_READ_QUEUE_CONCURRENCY,
-                AnomalyDetectorSettings.CHECKPOINT_WRITE_QUEUE_CONCURRENCY,
+                AnomalyDetectorSettings.AD_CHECKPOINT_READ_QUEUE_CONCURRENCY,
```

Comment:
Did you only rename ones to `AD_...` for settings that have a comparable forecasting one?

Reply:
Why did you not move all of them also to forecasting like CHECKPOINT_WRITE_QUEUE_MAX_HEAP_PERCENT. Are they either ones that wont have a comparable one in forecasting like maybe some of the preview ones or ones that you haven't needed in your other code yet?

Reply:
> Did you only rename ones to AD_... for settings that have a comparable forecasting one?
Yes